tests/conftest.py

```diff
@@ -7,20 +7,51 @@
 import pytest
 
 
+@pytest.fixture
```

Comment:
I tend to do this with fixtures always
https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session

```suggestion
@pytest.fixture(scope="session")
```

Reply:
No! That introduces a bug. This needs to be function-scoped (which is the default). I can put an explicit scoping if you like.

Reply:
Ah, sorry, wasn't paying attention. I like the sound of explicit scoping, for sure.